Assets/Scripts/CharacterController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CharacterController : MonoBehaviour {
+    // Raw hp amount, the player could have anywhere from 1-1000
+    float hp;
+    // This is going to be the current scale left on the hp bar, in proportion to how much raw health is left on the character
+    float hpScale;
+    // This is used to collect the intial raw scale value for the healthbar.  The healthbar could have a y value of like .9 so we want decutions to be made 
```

Comment:
What is this comment in reference to? Did you have a 'maxHp' variable before?

Reply:
I meant to indicate that hp would be the raw max hp of the character at start and would be decremented accordingly in a raw value.

Reply:
This has since been changed